src/libraries/icubmod/dragonfly2/common/DragonflyDeviceDriver2.cpp

```diff
@@ -40,6 +45,7 @@ int DragonflyDeviceDriver2Rgb::height () const
 	return RES(system_resources)->height();
 }
 
+#ifndef YARP_NO_DEPRECATED
```

Comment:
This is not YARP, you should not use `YARP_NO_DEPRECATED`. This is a device and not a library, so probably just removing the deprecated methods should be enough...